dependencies.yml

```diff
@@ -2,6 +2,9 @@
 # NB: Update NOTICE.txt and add/remove LICENSE.*.txt when adding/removing a dependency.
 #
 
+cglib:
+  cglib: { version: '3.2.10' }
```

Comment:
Perhaps we should update NOTICE.txt and add LICENSE.txt?

Reply:
I missed them. Thanks!